samples/build/build_buildah_cr.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 spec:
   source:
     url: https://github.com/sbose78/taxi
+    revision: c08014ffb7d1262c2674ea84db8a88742b00a2da
```

Comment:
Maybe add a comment `// alternatively, use "master"` to make it easy for folks to consume these samples with their own repos.

Reply:
I planed to use a better branch name (like `develop` branch), but I found you don't have a better branch in your repo.

I changed to use `master` as revision so that the end-user can understand what is it easily.

And I think in future, I think it is better move to use some common source repos :)

Reply:
> I think it is better move to use some common source repos

Absolutely! I didn't have a better example handy. Please feel free to propose a different source repo, or I'll do one anyway asap!